src/sdgio-edit.owl

```diff
@@ -3291,59 +3292,48 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00020303>)
 
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020304> (reformation of policies and institutions)
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020304> (reform policies and institutions)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020304> "A cross-cutting objective which specifies the objective of the planned processes undertaken by the CGIAR such that they reform policies and institutions so that such policies and institutions effectively contributes to achieving the CGIAR system level objectives.")
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020304> <http://orcid.org/0000-0062-3253-4387>)
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020304> "2019-05-09T00:36:14Z"^^xsd:dateTime)
-AnnotationAssertion(rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020304> "Research must provide evidence as a basis for reforming agri-food policies and institutions to make them more conducive to pro-poor development, to improved nutrition and the sustainable management of natural resources. Again, this spans most CGIAR and partner research activities with a special need in economic and social sciences.")
-AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020304> "reformation of policies and institutions"@en)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020304> "A cross-cutting objective which specifies the objective of the planned processes undertaken by the CGIAR such that they reform policies and institutions so that such policies and institutions effectively contributes to achieving the CGIAR system level objectives. As CGIAR notes: \"Research must provide evidence as a basis for reforming agri-food policies and institutions to make them more conducive to pro-poor development, to improved nutrition and the sustainable management of natural resources. Again, this spans most CGIAR and partner research activities with a special need in economic and social sciences.\"")
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020304> "reform policies and institutions"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00020304>)
 
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020305> (increment of agri-food research productivity and of innovation in the agri-food system)
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020305> (increase agri-food research productivity and of innovation in the agri-food system)
```

Comment:
Rephrase:

"increase research productivity and of innovation in the agri-food system"